packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataCmp.test.ts

```diff
@@ -101,21 +102,21 @@ describe('get metadata components path', () => {
 
   it('should return the path for a given username and metadata type', async () => {
     const metadataType = 'ApexClass';
-    expect(await cmpUtil.getComponentsPath(metadataType, alias)).to.equal(
+    expect(await cmpUtil.getComponentsPath(metadataType)).to.equal(
       expectedPath(metadataType)
     );
-    expect(getMetadataDirectoryPathStub.called).to.equal(true);
-    expect(getMetadataDirectoryPathStub.calledWith(username)).to.equal(true);
+    expect(metadataFolderStub.called).to.equal(true);
+    expect(metadataFolderStub.calledWith(username)).to.equal(false);
   });
 
   it('should return the path for a given folder', async () => {
     const metadataType = 'Report';
     const folder = 'TestFolder';
-    expect(
-      await cmpUtil.getComponentsPath(metadataType, alias, folder)
-    ).to.equal(expectedPath(metadataType + '_' + folder));
-    expect(getMetadataDirectoryPathStub.called).to.equal(true);
-    expect(getMetadataDirectoryPathStub.calledWith(username)).to.equal(true);
+    expect(await cmpUtil.getComponentsPath(metadataType, folder)).to.equal(
+      expectedPath(metadataType + '_' + folder)
+    );
```

Comment:
nit: I'd typically assign the values under test to variables here instead of inlining two calls (one async) in an expect.
```
const compPath = await cmpUtil.getComponentsPath(metadataType, folder);
const expectedPath = expectedPath(metadataType + '_' + folder);
expect(compPath).to.equal(expectedPath);
```
makes it easier to grok a failure and debug IMO